pkg/admissionpolicy/mutate_test.go

```diff
@@ -568,7 +570,25 @@ func Test_MutateResource(t *testing.T) {
 			resource, err := kubeutils.BytesToUnstructured(tt.rawResource)
 			assert.NilError(t, err)
 
-			response, err := mutateResource(policy, *resource)
+			gvk := resource.GroupVersionKind()
+			gvr := schema.GroupVersionResource{
+				Group:    gvk.Group,
+				Version:  gvk.Version,
+				Resource: strings.ToLower(gvk.Kind) + "s",
+			}
+
+			var client dclient.Interface = nil
+
+			namespaceSelectorMap := map[string]map[string]string{
+				"default": {
+					"env": "test",
+				},
+			}
+
+			isFake := true
```

Comment:
1. Why do we need a namespace selector? You can pass an empty one.
2. You can pass the `true` value directly to the function.